src/rules/noDuplicateSuperRule.ts

```diff
@@ -80,6 +80,11 @@ function walk(ctx: Lint.WalkContext<void>): void {
                     ? { node: node.parent! as ts.CallExpression, break: false }
                     : Kind.NoSuper;
 
+            case ts.SyntaxKind.ConditionalExpression: {
+                const { whenTrue, whenFalse } = node as ts.ConditionalExpression;
+                return worse(getSuperForNode(whenTrue), whenFalse !== undefined ? getSuperForNode(whenFalse) : Kind.NoSuper);
```

Comment:
`whenFalse` can't be undefined

Reply:
You're right of course. I'll fix it. There's another issue though. If someone would use nested ternary operators super() will be called twice:

```class Ternary extends Base {
	constructor(props?: any) {
        props ? super(props) : super() ? super() : super(); // if no props passed super() is called twice
	}
}
```
Do you think such a case should be covered? If so can you give me a tip how to handle it?